devel/STC/Portfile

```diff
@@ -31,7 +30,7 @@ configure.args-append \
 
 # Needs support for atomic operations.
 compiler.blacklist-append \
-                    *gcc-4.0 *gcc-4.2 {clang < 400}
+                    *gcc-4.0 *gcc-4.2 {clang}
```

Comment:
Rather than blacklist all versions of Xcode Clang, this should match the recommendation by upstream: So make this `{clang < 1200}`.

Reply:
> I thought they referred to LLVM versions. Sure, will update this today once on a desktop.

Well, Xcode Clang is based on the LLVM versions. Granted, there may be some differences relative to patches and such. But when it comes to C++ standards, I would hope Xcode Clang 12 would be equivalent to LLVM Clang 12.

So let's go with it for now. I can always expand the blacklisting once merged, if need be.